sdk/personalizer/Azure.AI.Personalizer/src/Generated/EvaluationClient.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.AI.Personalizer.Models;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.Personalizer
+{
+    /// <summary> The Evaluation service client. </summary>
+    public partial class EvaluationClient
```

Comment:
`EvaluationClient` seems a little bare when compared to `EvaluationsClient`.
Would users expect `Get(Async)()` and `Delete(Async)()` to be available on `EvaluationClient` itself, since those work on the individual `Evaluation` instances.

Reply:
Evaluation Client was created because we have an operation ID which is Evaluation_Apply. The autorest takes the operation ID and creates the Evaluation client. Applying an evaluation is an action on the service configuration so it is more fitting to be under the ServiceConfiguration client. I will change the operation ID so that the evaluation client is not created and the apply evaluation is under the service configuration client.